README.md

```diff
@@ -35,27 +40,64 @@ $ source ~/.zshrc
 
 5. Clone this git repository
 ```
-git clone https://github.com/shapirov103/ssp-eks-extension.git
-cd ssp-eks-extension
+git clone https://github.com/snyk-partners/snyk-monitor-eks-blueprints-addon.git
+cd snyk-monitor-eks-blueprints-addon
 ```
 
-6. Modify package.json and provide your name for the package including your organization. Example:
+## Installation
+The CDK code in this repository can be used to deploy a new Amazon EKS cluster with the Snyk Monitor built-in. Alternatively, the node module released from this repository can be used as a dependency in existing EKS Blueprints projects to integrate the Snyk Monitor.
+
+### New Amazon EKS Cluster
+1. Run `npm install`.
+
+2. Run `make build && make lint && make list` to build.
+
+3. Run `aws configure`. This command makes the `CDK_DEFAULT_ACCOUNT` and `CDK_DEFAULT_REGION` environment variables available to subsequent steps.
+
+4. Run `export INTEGRATION_ID=abcd1234-abcd-1234-abcd-1234abcd1234` with your actual integration ID. To get the integration ID, log in to your Snyk account and navigate to https://app.snyk.io/org/YOUR-ORGANIZATION-NAME/manage/integrations/kubernetes.
```

Comment:
You could chuck this into the README to help visualize where that integration ID comes from.
![image](https://user-images.githubusercontent.com/715967/162833536-3e653162-04bd-4d70-8cd4-8afb69bcad5d.png)